teal/legendary/formatter.tl

```diff
@@ -45,6 +49,10 @@ local function col1_str(item: LegendaryItem): string
 end
 
 local function col2_str(item: LegendaryItem): string
+  if vim.startswith(item.kind or '', 'legendary.function') or type((item as table)[1]) == 'function' then
+    return ''
```

Comment:
I wonder if it’s possible to get the name of the function somehow and put it here 🤔

Reply:
Probably not actually, since it might be an anonymous function without a name.

Reply:
Yes, I don't think it makes sense; I expect most of them to be anonymous. And even if wasn't, I don't think we can easily grab it (afaik, the autocommand api also doesn't support this, although it can show the call site).